VideoApp/VideoApp/TwilioVideo/RemoteParticipant/RemoteParticipantManager.swift

```diff
@@ -96,7 +96,11 @@ extension RemoteParticipantManager: RemoteParticipantDelegate {
         delegate?.participantDidChange(self)
     }
 
-    func remoteParticipantSwitchedOffVideoTrack(participant: RemoteParticipant, track: RemoteVideoTrack) {
+    func remoteParticipantSwitchedOffVideoTrack(
+        participant: RemoteParticipant,
+        track: RemoteVideoTrack,
+        reason: Track.SwitchOffReason
+    ) {
```

Comment:
For now I only had to update the delegate function interface. We were not previously checking disabled state for the camera track which I think was a bug. We should have been checking it and updating the UI when it was disabled. But now it just works with the new and improved track switch off API.

Reply:
In a later PR we will probably look at the switch off reason to distinguish in the UI when the user turned off the camera from when the server switched the track off. We are working on new UX designs for this.